packages/golden-layout/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@deephaven/golden-layout",
+  "version": "0.2.0",
+  "author": "Deephaven Data Labs LLC",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "cross-env": "^7.0.2",
+    "gulp": "^4.0.0",
+    "gulp-cli": "^2.3.0",
+    "gulp-concat": "^2.6.0",
+    "gulp-insert": "^0.5.0",
+    "gulp-uglify": "^1.5.3",
+    "karma": "0.13.*",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-cli": "^2.0.0",
+    "karma-jasmine": "~0.1.5",
+    "npm-run-all": "^4.1.5",
+    "rimraf": "^3.0.2",
+    "sass": "^1.32.13"
+  },
+  "description": "A multi-screen javascript Layout manager",
+  "main": "./dist/goldenlayout.js",
+  "types": "./index.d.ts",
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "jquery": "^3.5.1"
+  },
+  "scripts": {
+    "build": "cross-env NODE_ENV=production run-s clean build-steps",
+    "build-dev": "cross-env NODE_ENV=development run-s clean build-steps",
+    "build-steps": "run-p build:*",
+    "sass": "sass ./src/scss:./src/css -s compressed",
```

Comment:
Can this output to dist/css instead?

Reply:
Moved to dist/css and ignored the dist folder

Updated our imports of the css to use the dist folder